operator/v1/types_csi_cluster_driver.go

```diff
@@ -369,6 +369,21 @@ type VSphereCSIDriverConfigSpec struct {
 	// +openshift:enable:FeatureGate=VSphereDriverConfiguration
 	// +optional
 	GranularMaxSnapshotsPerBlockVolumeInVVOL *uint32 `json:"granularMaxSnapshotsPerBlockVolumeInVVOL,omitempty"`
+
+	// maxAllowedBlockVolumesPerNode is an optional configuration parameter that allows setting custom value for limit of
+	// PersistentVolumes attached to a node. In vSphere version 7 this limit was set to 59 by default, however in
+	// vSphere version 8 this limit was increased to 255. For more details see: https://configmax.broadcom.com/guest?vmwareproduct=vSphere&release=vSphere%208.0&categories=3-0
+	// Before increasing this value above 59 the cluster administrator needs to ensure that every node forming the
+	// cluster is updated to ESXi version 8 or higher and that all nodes are running the same version.
+	// Minimum allowed value: 1
+	// Maximum allowed value: 255 (matches the vSphere version 8 limit)
```

Comment:
We prefer to use full sentences for the documentation, so something like
```suggestion
	// The limit must be between 1 and 255, which matches the vSphere version 8 maximum.
```

Reply:
Committed and squashed - thanks.